orbit.py

```diff
@@ -1,9 +1,10 @@
 """Orbit sum main program."""
 
-from re import I
-from physicalobject import PhysicalObject
-import pygame
+from physicalobject_model import PhysicalObject_model
```

Comment:
Classes in Python are usually named using CamelCase, so:
```suggestion
from physicalobject_model import PhysicalObjectModel
```